spec/spec_helper.rb

```diff
@@ -31,6 +32,10 @@
 #
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
 RSpec.configure do |config|
+  
+  config.before(:each) do 
+    connection
```

Comment:
are you sure rspec is running? I have a report from your copy of this into my machine 
```
  NameError:
       undefined local variable or method `connect' for Database:Class
```